dags/utils/onepassword.py

```diff
@@ -0,0 +1,31 @@
+import os
+
+from onepasswordconnectsdk.client import Client, new_client
+import onepasswordconnectsdk
+
+ONEPASSWORD_CONNECT_HOST = os.getenv("OP_CONNECT")
+ONEPASSWORD_CONNECT_TOKEN = os.getenv("OP_API_TOKEN")
+VAULT_ID = os.getenv("OP_VAULT_ID")
+
+client: Client = new_client(ONEPASSWORD_CONNECT_HOST, ONEPASSWORD_CONNECT_TOKEN)
+
+
+def load_dict(required_secrets):
```

Comment:
i was initially going to suggest you make this function name more clear, like `load_secrets` - but i see you're proxying the `op` API: nice! 👍 🚢